core/modules/layout/layout.admin.inc

```diff
@@ -29,11 +29,15 @@ function layout_list_page() {
     $row = array();
     // Default layouts have a null path.
     if ($path === '') {
-      $row[] = array('data' => t('Default layouts (used on all paths without a specific layout)'), 'colspan' => 2);
+      $row[] = array('data' => t('Default layouts (used on all paths without a specific layout)'));
+      $row[] = '';
+      $row[] = '';
```

Comment:
Its to mitigate the problem you found at https://github.com/backdrop/backdrop/pull/1341#issuecomment-215006463 with the responsiveness of the title rows. I found that the mechanism for hiding columns based on priority tags in the header (eg `priority-low`) was negated by having `colspan` set on the row, and there is no way I could find to resize those rows with CSS as there is no CSS version of `colspan`. So instead, dont have one row that spans 3 columns, just have 3 colums, two of which are empty. The alternative is to resize with JS, but I thought this was simpler.